lib/ansible/modules/cloud/podman/podman_container.py

```diff
@@ -0,0 +1,1196 @@
+                cmd += ['--expose', exp]
+        if params.get('gidmap'):
+            cmd += ['--gidmap', params['gidmap']]
+        if params.get('group_add'):
+            cmd += ['--group-add', params['group_add']]
+        if params.get('healthcheck'):
+            cmd += ['--healthcheck', params['healthcheck']]
+        if params.get('healthcheck_interval'):
+            cmd += ['--healthcheck-interval', params['healthcheck_interval']]
+        if params.get('healthcheck_retries'):
+            cmd += ['--healthcheck-retries', str(params['healthcheck_retries'])]
+        if params.get('healthcheck_start_period'):
+            cmd += ['--healthcheck-start-period', params['healthcheck_start_period']]
+        if params.get('healthcheck_timeout'):
+            cmd += ['--healthcheck-timeout', params['healthcheck_timeout']]
+        if params.get('hostname'):
+            cmd += ['--hostname', params['hostname']]
+        if params.get('http_proxy') is not None:
+            cmd += ['--http-proxy=' + str(params['http_proxy'])]
+        if params.get('image_volume'):
+            cmd += ['--image-volume', params['image_volume']]
+        if params.get('init'):
+            cmd += ['--init', params['init']]
+        if params.get('init_path'):
+            cmd += ['--init-path', params['init_path']]
+        if params.get('interactive') is not None:
+            cmd += ['--interactive=' + str(params['interactive'])]
+        if params.get('ip'):
+            cmd += ['--ip', params['ip']]
+        if params.get('ipc'):
+            cmd += ['--ipc', params['ipc']]
+        if params.get('kernel_memory'):
+            cmd += ['--kernel-memory', params['kernel_memory']]
+        if params.get('label'):
+            for i in params.get('label'):
+                cmd += ["--label", i + "=" + params['label'][i]]
+        if params.get('label_file'):
+            cmd += ['--label-file', params['label_file']]
+        if params.get('log_opt'):
+            cmd += ['--log-opt', params['log_opt']]
+        if params.get('memory'):
+            cmd += ['--memory', params['memory']]
+        if params.get('memory_reservation'):
+            cmd += ['--memory-reservation', params['memory_reservation']]
+        if params.get('memory_swap'):
+            cmd += ['--memory-swap', params['memory_swap']]
+        if params.get('memory_swappiness'):
+            cmd += ['--memory-swappiness', str(params['memory_swappiness'])]
+        if params.get('mount'):
+            cmd += ['--mount', params['mount']]
+        if params.get('network'):
+            cmd += ['--network', params['network']]
+        if params.get('no_hosts') is not None:
+            cmd += ['--no-hosts=' + str(params['no_hosts'])]
+        if params.get('oom_kill_disable') is not None:
+            cmd += ['--oom-kill-disable=' + str(params['oom_kill_disable'])]
+        if params.get('oom_score_adj'):
+            cmd += ['--oom-score-adj', str(params['oom_score_adj'])]
```

Comment:
Instead of stringifying in place, you could just map str to all items in the final cmd. This'd probably make this all a bit cleaner.

Reply:
right, thanks for suggestion